src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const stylesArray = stylesString.split(';');
+  const stylesObject = {};
+
+  for (let i = 0; i < stylesArray.length; i++) {
+    const parts = stylesArray[i].split(':');
+
+    if (parts[0] && parts[1]) {
```

Comment:
You can simplify like this 
```suggestion
    const [property, value] = stylesArray[i].split(':').map(item => item.trim());

    if (property && value) {
```